packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -145,8 +144,146 @@ class InteractiveViewer extends StatefulWidget {
                  boundaryMargin.left.isFinite),
        ),
        constrained = false,
+       _transformChild = true,
        child = null;
 
+  /// Using `InteractiveViewer.raw` will prevent InteractiveViewer from transforming the
```

Comment:
I'm not sure on how the doc for this constructor flows, though. Can we have a single phrase  that indicates what this constructor does (at a glance) and the rest of this doc in a paragraph after?

See https://dart.dev/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary